lib/timescaledb/migrations/v01.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Migrations.V01 do
+  @moduledoc false
+
+  use Ecto.Migration
+
+  @spec up() :: any()
+  def up() do
+    create table(:peers_metrics, primary_key: {:id, :id, autogenerate: true}) do
+      add(:peer_id, :string, null: false)
+      add(:room_id, :string, null: false)
+      add(:"ice.binding_requests_received", :integer)
+      add(:"ice.binding_responses_sent", :integer)
+      add(:"ice.bytes_received", :integer)
+      add(:"ice.bytes_sent", :integer)
+      add(:"ice.packets_received", :integer)
+      add(:"ice.packets_sent", :integer)
+
+      timestamps(type: :utc_datetime_usec, updated_at: false)
+    end
+
+    create(index(:peers_metrics, [:inserted_at]))
+    create(index(:peers_metrics, [:peer_id]))
+    create(index(:peers_metrics, [:inserted_at, :peer_id]))
```

Comment:
Could you explain the need for the indices in a comment?

Reply:
Reason of adding these indices was to speed up most frequently executed queries to database